Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim-buster as base
+RUN apt-get update; apt-get install curl -y
```

Comment:
Subtle, but you might prefer to chain these commands with `&&` rather than `;` - the key difference is that if the `apt-get update` stage fails then it will run anyway with `;` (and I _believe_ it will carry on with later Docker stages) whereas with `&&` the command won't attempt to install curl and will fail there, which might help to make any errors more obvious

See [this SO discussion](https://unix.stackexchange.com/questions/187145/whats-the-difference-between-semicolon-and-double-ampersand) for a bit more depth